src/systems/systems/src/hydraulic/nose_steering.rs

```diff
@@ -155,7 +155,10 @@ impl SteeringActuator {
         pushback_tug: &impl Pushback,
     ) {
         if !pushback_tug.is_nose_wheel_steering_pin_inserted() {
-            self.update_max_speed(context, section_pressure.pressure());
+            self.update_max_speed(
+                context,
+                section_pressure.pressure_downstream_priority_valve(),
+            );
```

Comment:
I think you should pass on the `section_pressure: &impl SectionPressure` to the `SteeringActuator` call `pressure_downstream_priority_valve` on the actuator to unwrap instead of passing in the pressure directly.